### PR TITLE
OpenAPI Sync

### PR DESCRIPTION
**Pull request by cleb11**

### CODE REVIEW

File: openapi/mt_openapi_spec_v1.yaml [hunk 1]
```diff
@@ -16028,8 +16033,12 @@ components:
           "$ref": "#/components/schemas/foreign_exchange_rate"
           nullable: true
           description: Associated serialized foreign exchange rate information.
+        vendor_attributes:
+          type: object
+          nullable: true
+          description: 'Translation missing: en.openapi.descriptions.payment_order.schema.vendor_attributes'
```
Comment (cleb11) [inline openapi/mt_openapi_spec_v1.yaml:16039]: translation still missing

Comment (chen-annie) [reply openapi/mt_openapi_spec_v1.yaml:16039]: @rohanshiva Can you add in the description?

Comment (rohanshiva) [reply openapi/mt_openapi_spec_v1.yaml:16039]: Sorry! I didn't realize this was missing description. 

Does this work? https://github.com/Modern-Treasury/platform/pull/23704

Comment (cleb11) [reply openapi/mt_openapi_spec_v1.yaml:16039]: yah! thanks for adding

Comment (rohanshiva) [reply openapi/mt_openapi_spec_v1.yaml:16039]: Np! Will merge the PR right after the code freeze.

**Pull request closed without merging**